domain-models-interface-extensions/src/test/java/org/folio/rest/tools/PomReaderTest.java

```diff
@@ -88,6 +88,6 @@ void otherPom() throws IOException, XmlPullParserException {
     pom.init("src/test/resources/pom/pom-sample.xml");
     assertThat(PomReader.INSTANCE.getModuleName(), is("mod_inventory_storage"));
     assertThat(PomReader.INSTANCE.getVersion(), is("19.4.0"));
-    assertThat(PomReader.INSTANCE.getRmbVersion(), is("30.0.0"));
+    assertThat(PomReader.INSTANCE.getRmbVersion(), is("19.4.0"));
```

Comment:
Is `19.4.0` expected result? The pom-sample.xml actually has rmb version `30.0.0`

Reply:
Thanks for pointing out this discrepancy.
Now rmbVersion is always provided by maven in RmbVersion and never read from and pom.xml or properties files.